canonicalwebteam/search/views.py

```diff
@@ -71,13 +71,28 @@ def search_view():
                 num=num,
             )
 
-        return flask.render_template(
-            template_path,
-            query=query,
-            start=start,
-            num=num,
-            results=results,
-            siteSearch=site_search,
-        )
+            response = flask.make_response(
+                flask.render_template(
+                    template_path,
+                    query=query,
+                    start=start,
+                    num=num,
+                    results=results,
+                    siteSearch=site_search,
+                )
+            )
+            # nofollow, noindex
+            response.headers.set("X-Robots-Tag", "none")
```

Comment:
I think should should be `"noindex"` rather than `"none"`. `none` [is equivalent to](https://developers.google.com/search/docs/advanced/robots/robots_meta_tag#xrobotstag) `noindex, nofollow`. I don't think we want `nofollow` because if a search robot does happen to *parse* (without indexing) a search results page, we would probably like to discover the links in there and carry on crawling. General search bot coverage of our site is a good thing, we just don't want the results pages themselves to be indexed.